spec/ScorecardSpec.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+describe ("Scorecard", function() {
+
+  let scorecard;
+
+  beforeEach(function(){
+    scorecard = new Scorecard();
+  })
+
+  it ("should return the values of the first frames", () => {
```

Comment:
Really great test coverage, very detailed and you have made sure to test lots of different possible outcomes (i.e. for the tenth frame you have tested spare, one strike, two strikes etc).